app/points/domain/points-configuration.js

```diff
@@ -0,0 +1,39 @@
+const assertObjectType = require('./validation/assert-object-type')
+const assertNumber = require('./validation/assert-number')
+const assertBoolean = require('./validation/assert-boolean')
+const TierPointsConfiguration = require('./tier-points-configuration')
+
+class PointsConfiguration {
+  constructor (communityTierPointsConfig, licenseTierPointsConfig, custodyTierPointsConfig,
+              sdr, sdrConversion, nominalTargetSPO, defaultContractedHoursPO, defaultContractedHoursSPO, 
+              nominalTargetPO, paromsEnabled, parom) {
+    this.communityTierPointsConfig = communityTierPointsConfig
+    this.licenseTierPointsConfig = licenseTierPointsConfig
+    this.custodyTierPointsConfig = custodyTierPointsConfig
+    this.sdr = sdr
+    this.sdrConversion = sdrConversion
+    this.nominalTargetSPO = nominalTargetSPO
+    this.nominalTargetPO = nominalTargetPO
+    this.defaultContractedHoursPO = defaultContractedHoursPO
+    this.defaultContractedHoursSPO = defaultContractedHoursSPO
```

Comment:
We already have objects for the default contacted hours and default nominal targets in `app/points/domain` which could be passed into the constructor to make this a little simpler.